### PR TITLE
Clutch Widget Detection added

### PR DESCRIPTION
**Pull request by dhavalveera**
### technology requested, issue # [7203](https://github.com/wappalyzer/wappalyzer/issues/7203)

Vendor Site - https://clutch.co/

Category - Reviews, Widgets

**Example Sites:**

https://bluesoft.com/
https://www.xfive.co/

### CODE REVIEW

**Review by dhavalveera — COMMENTED** [review]
#7203

File: src/technologies/c.json [hunk 1]
```diff
@@ -2139,6 +2139,16 @@
...
     "website": "https://clustrmaps.com/"
   },
+  "Clutch": {
+    "cats": [
+      5
+    ],
+    "description": "Ratings and reviews of leading IT, marketing, and business services companies. Clutch is your data-driven field guide for B2B buying and hiring decisions.",
+    "dom": "div[class*='clutch-widget'], div[data-url*='https://widget.clutch.co']",
+    "icon": "clutch.png",
+    "scriptSrc": "https://widget.clutch.co/static/js/widget.js",
```
Comment (nurbek91) [inline src/technologies/c.json:2149]: `"scriptSrc":  //widget\\.clutch\\.co/`

File: src/technologies/c.json [hunk 2]
```diff
@@ -2139,6 +2139,16 @@
     "scriptSrc": "clustrmaps\\.com",
     "website": "https://clustrmaps.com/"
   },
+  "Clutch": {
+    "cats": [
+      5
+    ],
+    "description": "Ratings and reviews of leading IT, marketing, and business services companies. Clutch is your data-driven field guide for B2B buying and hiring decisions.",
```
Comment (nurbek91) [inline src/technologies/c.json:2146]: `"description": "Clutch review widgets are stand-alone applications that you can embed on your website to show your dynamic ratings and reviews.",`

File: src/technologies/c.json [hunk 3]
```diff
@@ -2139,6 +2139,16 @@
...
+    "dom": "div[class*='clutch-widget'], div[data-url*='https://widget.clutch.co']",
```
Comment (nurbek91) [inline src/technologies/c.json:2147]: This line can be removed

File: src/technologies/c.json [hunk 4]
```diff
@@ -2139,6 +2139,16 @@
...
+    "icon": "clutch.png",
```
Comment (nurbek91) [inline src/technologies/c.json:2148]: svg
Clutch

File: src/technologies/c.json [hunk 5]
```diff
@@ -2139,6 +2139,16 @@
...
+    "scriptSrc": "https://widget.clutch.co/static/js/widget.js",
+    "website": "https://clutch.co/"
```
Comment (nurbek91) [inline src/technologies/c.json:2150]: `"website": "https://clutch.co/content/add-review-widget-your-website"`

File: src/technologies/c.json [hunk 6]
```diff
@@ -2139,6 +2139,16 @@
     "scriptSrc": "clustrmaps\\.com",
     "website": "https://clustrmaps.com/"
   },
+  "Clutch": {
+    "cats": [
+      5
+    ],
```
Comment (nurbek91) [inline src/technologies/c.json:2145]: 90
5

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR.Need to improve it

File: src/technologies/c.json (hunk 1 above)
Comment (dhavalveera) [reply src/technologies/c.json:2149]: done

File: src/technologies/c.json (hunk 3 above)
Comment (dhavalveera) [reply src/technologies/c.json:2147]: `"description"` needs to be removed?

File: src/technologies/c.json (hunk 4 above)
Comment (dhavalveera) [reply src/technologies/c.json:2148]: done

File: src/technologies/c.json (hunk 2 above)
Comment (dhavalveera) [reply src/technologies/c.json:2146]: done

File: src/technologies/c.json (hunk 5 above)
Comment (dhavalveera) [reply src/technologies/c.json:2150]: one

File: src/technologies/c.json (hunk 6 above)
Comment (dhavalveera) [reply src/technologies/c.json:2145]: done

**Review by nurbek91 — APPROVED** [review]

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**